api/v1alpha1/flowcollector_webhook.go

```diff
@@ -58,7 +58,33 @@ func (r *FlowCollector) ConvertTo(dstRaw conversion.Hub) error {
 		dst.Spec.Processor.Metrics.DisableAlerts = restored.Spec.Processor.Metrics.DisableAlerts
 	}
 
-	dst.Spec.Loki.StatusTLS = restored.Spec.Loki.StatusTLS
+	dst.Spec.Loki.Manual.URL = restored.Spec.Loki.Manual.URL
```

Comment:
just wondering, would `dst.Spec.Loki.Manual = restored.Spec.Loki.Manual` just work for all of that?

Reply:
I think so, but not completely sure. I'm checking on that.

Reply:
Finally getting this one. Yeah, it seems obvious this one and I honestly don't remember why I made all fields separately. Just changed. Thanks for catching that one.